packages/component-library/stories/Checkbox.story.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
If you remove this line, does it cause any linting errors? If so, where?

Reply:
When I remove the lines about eslint, that are already commented out, I get a message in VS Code:
`@emotion/core should be listed in the project's dependencies` in Checkbox.js, and a similar message in Checkbox.story.js about storybook should be in the project's dependencies.